packages/@o3r/store-sync/package.json

```diff
@@ -23,6 +23,7 @@
     "@angular-devkit/schematics": "~16.2.0",
     "@ngrx/entity": "~16.3.0",
     "@ngrx/store": "~16.3.0",
+    "@ngrx/store-devtools": "~16.3.0",
```

Comment:
it has to be optional peer if it's only for typing

Reply:
Noted